modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -81,7 +82,8 @@
 					"liferay-ui:input-editor:allowBrowseDocuments", true
 				).put(
 					"liferay-ui:input-editor:name",
-					ddmFormFieldRenderingContext.getName()
+					StringUtil.replace(
+						ddmFormFieldRenderingContext.getName(), '$', '_')
```

Comment:
@linolaoj, why are you saying on the commit message "avoiding editor name to be escaped"?

Reply:
Hi @natocesarrego , the Editor name parameter is being escaped. The char '$' present in the ddmFormField name turns into something like '%%24'. The issue happens because the editor sends events with the escaped name. The idea of this commit is to use '_' to avoid the editor name being escaped.

Reply:
Understood! Thanks for the explanation, @linolaoj. :wink: